.github/workflows/ci.yml

```diff
@@ -41,25 +41,14 @@ jobs:
           override: true
           components: rustfmt, clippy
       - name: Check with Rustfmt
-        uses: actions-rs/cargo@v1
-        with:
-          command: fmt
-          args: --all -- --check
+        run: cargo fmt --all --check
       - name: Check with Clippy
-        uses: actions-rs/clippy-check@v1
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-          args: --all-targets ${{ matrix.features }} -- -D warnings
+        run: cargo clippy
```

Comment:
I think clippy should get "--all-targets" and "${{ matrix.features }}" otherwise it might skip tests and other code.

Reply:
Oops, not sure how I lost that :grimacing:.

Reply:
Fixed, but now CI fails because we have a ton of clippy warnings.

Reply:
Filed #326. It doesn't actually fail (yet), I was using `beta`.